site/en/docs/extensions/mv3/permission_warnings/index.md

```diff
@@ -349,15 +337,9 @@ the steps in [Viewing Warnings][27].
 
 ## Update permissions {: #update_permissions }
 
-Updating an extension with additional permissions may temporarily disable it. The user will have to
+Updating an extension with additional permissions with warnings may temporarily disable it. The user will have to
```

Comment:
I'm not clear on what 'with warnings' adds to this sentence. Please explain it to me and I might have a suggestion.

Reply:
@jpmedley 
Only permissions that "**trigger a warning**" (those listed on the [table](https://deploy-preview-4668--developer-chrome-com.netlify.app/docs/extensions/mv3/permission_warnings/#permissions_with_warnings)), can potentially disable an extension.

Other permissions that don't trigger a warning (like storage, runtime, etc) will never disable the extension.

Reply:
That clarifies it. Can you replace the one sentence with the two you just wrote (or something similar). Then we'll be done.